app/javascript/src/main.js

```diff
@@ -0,0 +1,15 @@
+import { createApp } from 'vue'
+import Chat from '@/components/Chat.vue'
+import Message from '@/components/Message.vue'
+
+export default () => {
+    document.addEventListener('DOMContentLoaded', () => {
+        const app = createApp(Chat)
+        app.mount('#app-chat')
+    })
+
+    document.addEventListener('DOMContentLoaded', () => {
+        const app = createApp(Message)
+        app.mount('#app-message')
+    })
```

Comment:
why did you decide to go with 2 vue apps? I thought it would be one app

Reply:
@goodniceweb meaningless code and I have placed just one app